aten/src/ATen/CMakeLists.txt

```diff
@@ -63,7 +63,7 @@ FILE(GLOB native_quantized_cpp
 FILE(GLOB native_cuda_cu "native/cuda/*.cu")
 FILE(GLOB native_cuda_cpp "native/cuda/*.cpp")
 FILE(GLOB native_cudnn_cpp "native/cudnn/*.cpp")
-FILE(GLOB native_sparse_cuda_cu "native/sparse/cuda/*.cu")
+FILE(GLOB native_sparse_cuda_cu "native/sparse/cuda/*.cu" "native/sparse/cuda/*.cuh")
```

Comment:
We intentionally don’t include the headers here.

Reply:
Ah, the context here was Aapo was wondering why cmake wasn't recompiling when headers were modified. So the problem is we probably need some `native_sparse_cuda_h` variable and need to properly declare it as a dependency.

Reply:
Ok let me remove that then?

Reply:
But for example the `hip_h` rule does include .cuhs ?

Reply:
@akyrola `hip_h` are all header files (the equivalent for cuda is `cuda_h` above), while here `native_sparse_cuda_cu` represents the cu compilation units.

Reply:
ok let me remove